docker/windows/Scripts/configure-and-run-tentacle-wrapper.ps1

```diff
@@ -0,0 +1,2 @@
+﻿& ./configure-tentacle.ps1 -Verbose
```

Comment:
Am I right in remembering that the `&` here makes the script run in the background?

Reply:
in powershell it seems `&` at the start means "call a process" and `&` at the end means run in the background. How confusing!
https://ss64.com/ps/call.html
https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_operators?view=powershell-7.3&viewFallbackFrom=powershell-7.1#call-operator-
https://stackoverflow.com/questions/22074507/what-does-the-symbol-in-powershell-mean#:~:text=%26%20is%20the%20call%20operator%20which,a%20script%2C%20or%20a%20function.

Reply:
Ah right, yeah that could take a bit to get used to. My potential concern was the two scripts not running in sequence anymore, but seems fine if the `&` at the start just means "run this process and wait please".

Reply:
The tentacle tests do run the windows tentacle docker container, so if it is completely broken the test chain should catch it.